Composer/packages/types/src/indexers.ts

```diff
@@ -212,3 +213,5 @@ export interface BotProjectFile {
   content: BotProjectSpace;
   lastModified: string;
 }
+
+export type RecognizerType = 'DefaultRecognizer' | 'RegexRecognizer' | 'CustomRecognizer';
```

Comment:
I think that this type is not necessary here, we can use $kind as the constant which is compatible to the custom recognizer scenario